docs/roadmap.rst

```diff
@@ -0,0 +1,111 @@
+##########################################
+Rubin Science Platform Status and Roadmap
+##########################################
+
+What can you do with the Rubin Science Platform?
+================================================
+
+Below you will find a detailed (but not exhaustive) list of current and future RSP capabilities.
```

Comment:
Consider dropping ths sentence because it doesn't relate to the current section. Perhaps move this up to a lead paragraph below the document title that describes the purpose of the page.